gcloud-java-resourcemanager/src/test/java/com/google/gcloud/resourcemanager/PolicyTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.resourcemanager;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import com.google.gcloud.resourcemanager.Policy.Binding;
+import com.google.gcloud.resourcemanager.Policy.Member;
+import com.google.gcloud.resourcemanager.Policy.RoleType;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PolicyTest {
+
+  private static final Binding OWNER_BINDING;
+  private static final Binding EDITOR_BINDING;
+  private static final Binding VIEWER_BINDING;
+  private static final Binding EMPTY_BINDING = Policy.Binding.builder().build();
+  private static final List<Member> OWNER_MEMBER_LIST = new ArrayList<>();
+  private static final List<Member> EDITOR_MEMBER_LIST = new ArrayList<>();
+  private static final List<Member> VIEWER_MEMBER_LIST = new ArrayList<>();
+  static {
+    OWNER_MEMBER_LIST.add(Member.user("first-owner@email.com"));
+    OWNER_MEMBER_LIST.add(Member.group("group-of-owners@email.com"));
+    OWNER_BINDING =
+        Policy.Binding.builder().role(RoleType.OWNER).members(OWNER_MEMBER_LIST).build();
+    EDITOR_MEMBER_LIST.add(Member.serviceAccount("editor@someemail.com"));
+    EDITOR_BINDING =
+        Policy.Binding.builder().role(RoleType.EDITOR).members(EDITOR_MEMBER_LIST).build();
+    VIEWER_MEMBER_LIST.add(Member.serviceAccount("app@someemail.com"));
+    VIEWER_MEMBER_LIST.add(Member.user("viewer@email.com"));
+    VIEWER_BINDING =
+        Policy.Binding.builder().role(RoleType.VIEWER).members(VIEWER_MEMBER_LIST).build();
+  }
```

Comment:
What about using [ImmutableList.of](http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/collect/ImmutableList.html#of()) instead?